pom.xml

```diff
@@ -109,15 +109,16 @@
     <properties>
         <apt.phase>process-sources</apt.phase>
         <mule.version>4.1.1</mule.version>
-        <mule.weave.version>2.1.2</mule.weave.version>
+        <mule.impl.version>4.1.6-20220221</mule.impl.version>
+        <mule.weave.version>2.1.2-20220125</mule.weave.version>
         <mule.sdk.version>1.1.5</mule.sdk.version>
         <mule.api.version>1.1.1</mule.api.version>
         <mule.metadata.version>1.1.1</mule.metadata.version>
         <mule.extensions.ast.loader.version>1.0.1</mule.extensions.ast.loader.version>
         <mule.extensions.maven.plugin.version>1.1.6</mule.extensions.maven.plugin.version>
         <mule.app.plugins.maven.plugin.version>1.1.1</mule.app.plugins.maven.plugin.version>
 
-        <guava.version>25.1-jre</guava.version>
+        <guava.version>32.0.0-android</guava.version>
```

Comment:
Why an Android Version?

Reply:
My bad. It was the suggestion from dependabot, but there is a `jre` version available so changing it.